workshops/build-a-binderhub/workshop-presentations/zero-to-binderhub.md

```diff
@@ -293,6 +350,8 @@ kubectl patch deployment tiller-deploy \
     }]'
 ```
 
+**NOTE:** I would recommend removing the line breaks and whitespace from this command.
```

Comment:
Could you add why? If it's something people should do, is it worth providing a copy-pastable version of the command as well?